### ISSUE TITLE
Error while opening excel

### ISSUE THREAD

**Issue by jansor** [openspout/openspout#90]
Hi, Just noticed this.

I have loaded the excemple script and just added one row as per script below

```php
include_once 'vendor/autoload.php';

use OpenSpout\Common\Entity\Cell;

$writer = new \OpenSpout\Writer\XLSX\Writer();
$writer->openToBrowser('testcarl.xlsx');
$cells = [
    Cell::fromValue('Carl'),
    Cell::fromValue('is'),
    Cell::fromValue('great!'),
];

$writer->close();`
```

After i download the file, and try opening in excel, following issue arrise:
![image](https://user-images.githubusercontent.com/1769830/176863770-ff3c99ee-e1fc-47c8-9223-506f15d68720.png)

I have tested this on PHP 8.1.3 and 8.1.7 clean php versions
Anyone have same issues like this or where do i need to look for changes?

When i reverted back to the original code from Box Sprout it was working correctlly?

**Comment by Slamdunk** [comment]
Hi, the provided snippet doesn't write any cell, since the `$cells` variable is not used.

**Comment by jansor** [comment]
Ow that must have happend when i clean up the code, even with adding the cell it still provides an issue, 
Beside that, even when you dont add any cell, it still should not provide that can of error, i mean, i would only get empty workbook

 Here is the result even when adding one row:
![image](https://user-images.githubusercontent.com/1769830/176877328-372d2e81-62d1-425c-8d24-588b36ce0fe8.png)

**Comment by Slamdunk** [comment]
I'm running Microsoft Office 2021 and with this fixed snipped:

```php
include_once 'vendor/autoload.php';

use OpenSpout\Common\Entity\Cell;

$writer = new \OpenSpout\Writer\XLSX\Writer();
$writer->openToBrowser('testcarl.xlsx');
$writer->addRow(\OpenSpout\Common\Entity\Row::fromValues([
    'Carl',
    'is',
    'great!',
]));

$writer->close();
```

I don't have any popups after opening the file.

May you report here the log file please?

**Comment by jansor** [comment]
Did some additional testing, i can see that their are additional dependencies needed betweeen Box spout and open spout, but those are direct from PHP itself, sow i assume that cannot be ther problem.
Also tested this under NGINX linux and windows, same issues happens.

Only thing i can see when open xlsx as zip file is as per below:
![image](https://user-images.githubusercontent.com/1769830/176996995-bd88d919-345e-4b2d-be44-650c9b49acab.png)

Where the left one is not possible to open
[banana (1).xlsx](https://github.com/openspout/openspout/files/9032839/banana.1.xlsx)


In the mean time i have also setup new vm and tested their the test code and that seems to work.
Sow pretty sure its a config setting on my side, and the bug is not related with the openspout as package.

Only question now is what config issue??? Their goes my saterday working......ugh

**Comment by jansor** [comment]
Minor update, when i remove xl syles.xml and xl workbook.xml out of the zip archive, and try to open the excel file again, it is working. any idee, or suggestuon where i can look for this?

**Comment by Slamdunk** [comment]
Which os are you running php onto?

**Comment by jansor** [comment]
At this moment for testing i user windows 10 with IIS and PHP8.1.7 with latest version of openspout 4.2.6

**Comment by jansor** [comment]
For today i give up, tomorrow i will test again under NGINX to se if it also happens, but at this moment, still have the issue.

**Comment by Slamdunk** [comment]
Hi, I guess the culprit is Windows then.
We already had an issue related to it: https://github.com/openspout/openspout/pull/87
Maybe somewhere we are meeting the directory separator char

**Comment by AliHashemzadeh** [comment]
Hello,

I have same problem on Windows 10 using xampp with PHP 8.1.6 and Laravel 8.
`
$writer = new \OpenSpout\Writer\XLSX\Writer();
$writer->openToBrowser('test.xlsx');
$writer->addRow(\OpenSpout\Common\Entity\Row::fromValues(['foo', 'bar', 'baz']));
$writer->close();
`

**Comment by AliHashemZadeh** [comment]
Hi,

I have same problem using example in docs with Windows 10 with xampp, PHP 8.1.6 and Laravel 8.
```php
$writer = new \OpenSpout\Writer\XLSX\Writer();

$writer->openToBrowser('test.xlsx');
$writer->addRow(\OpenSpout\Common\Entity\Row::fromValues(['foo', 'bar', 'baz']));

$writer->close();
```

**Comment by spackmat** [comment]
Hey, I have the same problem (Windows 11, plain PHP 8.1.2 through Symfony local server, openspout/openspout 4.6.2).

I patched the `src/Writer/Common/Helper/ZipHelper.php` according to the change in #87 and that fixes the problem for me. Can confirm that the zip file created by the patched helper doesn't contain those weird named files on root anymore and Excel opens the file without complaining.

So all we need is a new release containing that PR. :)

**Comment by Slamdunk** [comment]
I've released #87 in [`v4.7.0`](https://github.com/openspout/openspout/releases/tag/v4.7.0), please confirm this bug is gone so we can close the issue

**Comment by spackmat** [comment]
@Slamdunk That was quick! I Upgraded to v4.7.0 and can confirm the created file is clean and Excel does not complain anymore. Thanks!

**Comment by AliHashemZadeh** [comment]
> I've released #87 in [`v4.7.0`](https://github.com/openspout/openspout/releases/tag/v4.7.0), please confirm this bug is gone so we can close the issue

I upgraded to v4.7.0 and problem was solved. Thanks.
